sdk/core/core-http/review/core-http.api.md

```diff
@@ -26,9 +26,9 @@ export interface AccessTokenCache {
 
 // @public
 export class AccessTokenRefresher {
-    constructor(credential: TokenCredential, scopes: string | string[], requiredMillisecondsBeforeNewRefresh?: number);
-    isReady(): boolean;
+    constructor(credential: TokenCredential, scopes: string | string[], tokenRefreshIntervalInMs?: number);
     refresh(options: GetTokenOptions): Promise<AccessToken | undefined>;
+    startIfReady(options: GetTokenOptions, cache?: ExpiringAccessTokenCache): void;
```

Comment:
Not a fan of this name, it's not really clear what it's doing.

Also I wonder if the second arg should be an optional options bag, in case there are later optional args added. 

Lastly, why is `cache` an `ExpiringAccessTokenCache` instead of just an `AccessTokenCache`? Seems like it should take the generic interface instead of a specific class implementation

Reply:
Resolving since I reverted this.